README.md

```diff
@@ -1,97 +1,82 @@
-
-| Parameter | Is Required | Default | Description           |
-|-----------|-------------|---------|-----------------------|
-| `input`   | true        |         | Description goes here |
-
-## Outputs
+When the action runs it will add a deployment and deployment status record to the repo.  The GitHub Deployment will also have a special task d
 
-| Output   | Description           | Possible Values |
-|----------|-----------------------|-----------------|
-| `output` | Description goes here |                 |
+## Inputs
 
-## Usage Examples
+| Parameter        | Is Required | Description                                                                                                                                                                                         |
+| ---------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `workflow-actor` | true        | The GitHub user who triggered the workflow                                                                                                                                                          |
+| `token`          | true        | A token with `repo_deployment` permissions to create and update issues, workflows using this action should be granted `permissions` of `deployments: write`                                         |
+| `environment`    | true        | The environment the release was deployed to, i.e. [Dev\|QA\|Stage\|Demo\|UAT\|Prod]                                                                                                                 |
+| `entity`         | true        | The entity that is deployed, i.e. "proj-app", "proj-infrastruction" or "proj-db"                                                                                                                    |
+| `instance`       | true        | A freeform identifier to distinguish separately deployed instances of the entity in the same environment. Typical uses would be to name a slot and/or region, e.g "NA26", "NA26-slot1", "NA27-blue" |
+
+## Usage Example
+
+```yaml
+name: Take Down Testing
+on:
+  workflow_dispatch:
+    environment:
+      description: The testing environment
+      required: true
+      type: choice
+      options:
+        - Dev
+        - QA
+    inputs:
+      instance:
+        description: The instance to deploy to
+        required: true
+        type: choice
+        options:
+          - Primary-Test-Slot1
+          - Primary-Test-Slot2
+          - Secondary-Test-Slot1
+          - Secondary-Test-Slot2
+
+# Permissions needed to add GitHub Deployment and Deployment
+# status objects
+permissions:
+  deployments: write
 
-```yml
 jobs:
-  jobname: # TODO: fix the job name
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v3
+  environment: ${{ github.event.inputs.environment }}
```

Comment:
Does this work? I didn't know you could put the `environment` here.

Reply:
You can if you squint your eyes and accidentally have a line move up by your editor.